README.md

```diff
@@ -1,2 +1,131 @@
-# termviz
-ROS visualisation on the terminal
+# TermViz - ROS visualization on the terminal
+
+TermViz is a visualizer for the Robot Operating System (ROS) framework, that runs completely on the terminal.
+This allows the visualizer to run directly on the robot (e.g., via ssh) without any configuration on an external computer.
+It can also be useful in case the robot has a firewall or in situations in which no graphics server can be run.
+
+## How to use
+
+To launch the visualizer, just run the `termviz` executable.
```

Comment:
how to get it?

Reply:
Will include this in the Installation section ;)